engine/gui/src/gui.cpp

```diff
@@ -1477,17 +1499,16 @@ Result DeleteDynamicTexture(HScene scene, const dmhash_t texture_hash)
             }
         }
 
+        if (num_pruned)
+        {
+            std::sort(c->m_RenderNodes.Begin(), c->m_RenderNodes.End(), RenderEntrySortDisabledPred());
```

Comment:
Sorting these tiny structs really takes no time at all.
We move all the renderkey=0xFFFFFFFF to the end of the array.